coil/src/androidTest/java/dev/chrisbanes/accompanist/coil/CoilTest.kt

```diff
@@ -175,6 +176,39 @@ class CoilTest {
         loadCompleteSignal.close()
     }
 
+    @OptIn(ExperimentalCoroutinesApi::class)
+    @Test
+    fun basicLoad_changeSize() {
+        val loadCompleteSignal = Channel<Unit>(Channel.UNLIMITED)
+        val sizeFlow = MutableStateFlow(128.dp)
```

Comment:
What's the lifecycle of this `MutableStateFlow` you cannot cancel / stop / cancel it manually. What happens when the test finishes?

Reply:
`StateFlow` doesn't have a lifecycle, it's just an observable value store. After the test finishes it just gets garbage collected I guess.

Reply:
SG, was just checking that you thought about this